config/config_test.go

```diff
@@ -25,8 +25,8 @@ func TestSpec(t *testing.T) {
 				So(cfg.DownloadServiceToken, ShouldEqual, "QB0108EZ-825D-412C-9B1D-41EF7747F462")
 				So(cfg.FilterAPIURL, ShouldEqual, "http://localhost:22100")
 				So(cfg.GracefulShutdownTimeout, ShouldEqual, 5*time.Second)
-				So(cfg.HealthCheckInterval, ShouldEqual, 1*time.Minute)
-				So(cfg.HealthCheckRecovery, ShouldEqual, 5*time.Second)
+				So(cfg.HealthCheckInterval, ShouldEqual, 10*time.Second)
+				So(cfg.HealthCheckCriticalTimeout, ShouldEqual, 1*time.Minute)
 				So(cfg.VaultToken, ShouldEqual, "")
 				So(cfg.VaultPath, ShouldEqual, "secret/shared/psk")
 				So(cfg.ServiceAuthToken, ShouldEqual, "c60198e9-1864-4b68-ad0b-1e858e5b46a4")
```

Comment:
I noticed that this tests assumes that no env var for the config is set, which might not be true (e.g. setting env var IS_PUBLISHING=false, and then run `make test` would cause the test to fail for the wrong reason). I will fix this test too.